ext_mongodb.php

```diff
@@ -200,7 +200,7 @@ function($value) {
 		if ($this->writeConcern) {
 			$ret['writeConcern'] = $this->writeConcern;
 		} else {
-			$ret['writeConcern'] = NULL;
+			$ret['writeConcern'] = [];
```

Comment:
Yes, NULL is better here I think. I'll change PHPC in the acompanying branch/PR https://github.com/mongodb/mongo-php-driver/pull/416.